tools/fp20compiler/main.cpp

```diff
@@ -84,6 +146,11 @@ int main(int argc, char** argv) {
     fread(buffer, size, 1, fh);
     buffer[size] = '\0';
 
+    if (strchr(buffer, '\r') != NULL) {
+        fprintf(stderr, "found unsupported line-endings\n");
+        exit(1);
+    }
```

Comment:
On Linux, `\r\n` will fail with fp20compiler suddenly trying to read from `stdin`. But on Windows it probably works fine? Or maybe cgc &rarr; fp20compiler is broken on Windows?